src/Components/TwoOneOneComponents/TwoOneOneHeader/NCTwoOneOneHeader.tsx

```diff
@@ -0,0 +1,201 @@
+import { useContext, useState, useEffect } from 'react';
+import { FormattedMessage, useIntl } from 'react-intl';
+import { AppBar, MenuItem, Select, Link, IconButton, Dialog, SelectChangeEvent } from '@mui/material';
+import Stack from '@mui/material/Stack';
+import Box from '@mui/material/Box';
+import Paper from '@mui/material/Paper';
+import { useConfig } from '../../Config/configHook';
+import { Context } from '../../Wrapper/Wrapper';
+import NCtwoOneOneMFBLogo from '../../../Assets/States/NC/WhiteLabels/TwoOneOneAssets/nc211-MFB-CTD-2025.png';
+import twoOneOneNCLinks from '../../../Assets/States/NC/WhiteLabels/TwoOneOneAssets/twoOneOneNCLink';
+import LanguageIcon from '@mui/icons-material/Language';
+import ShareIcon from '@mui/icons-material/Share';
+import MenuIcon from '@mui/icons-material/Menu';
+import CloseIcon from '@mui/icons-material/Close';
+import TwoOneOneShare from '../TwoOneOneShare/TwoOneOneShare';
+import './TwoOneOneHeader.css';
+
+type LanguageOptions = {
+  [key: string]: string;
+};
+
+const NCTwoOneOneHeader = () => {
```

Comment:
I suggest to keep naming consistent and easier to read, put "nc" after "twooneone", "TwoOneOneNCHeader"